### ISSUE TITLE
Two start tags in sequence move inner end tag to the end of the outer one after round trip

### ISSUE THREAD

**Issue by pchupik** [commonsguy/cwac-richedit#15]
If we have 
```html
<b><i>The quick brown fox</i> jumped over the lazy dog.</b>
```
after fromXhtml( ) and back toXhtml( ) result will be
```html
<i><b>The quick brown fox jumped over the lazy dog.</b></i>
```
It is expected result to be same as initial input.

**Comment by commonsguy** [comment]
Yes, I can reproduce this problem. I will work on fixing it -- thanks!

**Comment by commonsguy** [comment]
Fixed in 0.5.1. Thanks!
